api-reference/docstubs/api/user-reminderview.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "reminderView"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# reminderView
+
+Namespace: microsoft.graph
+
+
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Determine scopes **|
+|Delegated (personal Microsoft account)|Not supported.|
+|Application|**TODO: Determine AppOnly scopes **|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /me/reminderView
+GET /users/{usersId}/reminderView
+GET /me/managedDevices/{managedDeviceId}/users/{userId}/reminderView
```

Comment:
Line 33: does reminderView apply to managedDevice in Intune? Intune doesn't include this in their part of the TOC, so maybe not?